record_indexer/create_collection_index.py

```diff
@@ -0,0 +1,123 @@
+import argparse
+from datetime import datetime
+import json
+import os
+import sys
+
+import boto3
+import requests
+
+from .add_page_to_index import add_page
+from . import settings
+
+
+def update_alias_for_collection(alias: str, collection_id: str, index: str):
+    remove_collection_indices_from_alias(alias, collection_id)
+
+    url = f"{settings.ENDPOINT}/_aliases"
+    headers = {"Content-Type": "application/json"}
+
+    data = {"actions": [{"add": {"index": index, "alias": alias}}]}
+
+    r = requests.post(url, headers=headers, data=json.dumps(data), auth=settings.AUTH)
+    r.raise_for_status()
+    print(f"added index `{index}` to alias `{alias}`")
+
+
+def remove_collection_indices_from_alias(alias: str, collection_id: str):
+    url = f"{settings.ENDPOINT}/_alias/{alias}"
+    r = requests.get(url=url, auth=settings.AUTH)
+    if r.status_code == 404:
+        return
+    else:
+        r.raise_for_status()
+        indices = json.loads(r.text)
```

Comment:
Throughout, I notice you use `json.loads(r.text)`, so I just thought I'd mention that `r.json()` is a [built in requests feature](https://requests.readthedocs.io/en/latest/user/quickstart/#json-response-content) to do the same thing. No reason to necessarily switch, but just fyi

Reply:
Oh yeah I was somehow unaware of this! Thanks for mentioning 👍